com.heroicgameslauncher.hgl.metainfo.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.heroicgameslauncher.hgl</id>
+  
+  <name>Heroic Games Launcher</name>
+  <summary>An Open Source GOG and Epic Games Launcher. </summary>
+  <url type="homepage">https://heroicgameslauncher.com</url>
```

Comment:
Consider adding the following extra URL fields:
 * `<url type="bugtracker">...</url>`
 * `<url type="donation">...</url>`
 * `<url type="translate">...</url>`